docs/source/guides/scargo-esp32.rst

```diff
@@ -38,4 +38,11 @@ It will generate build/flash_image.bin file. This file can be used with a Quick
 
 Building a project
 ------------------
-The project can be built using both scargo build of idf.py build commands.
\ No newline at end of file
+The project can be built using both scargo build of idf.py build commands.
+
+Configure ESP32 project
+------------------------
+To configure your project for chosen ESP32 chipset, usage please change it in scargo.toml file in **[esp32]** section
```

Comment:
I guess this line is not needed. There is "It's also possible to change ... " line below.

Reply:
fixed